include/alpaka/event/EventCpu.hpp

```diff
@@ -509,16 +530,25 @@ namespace alpaka
                     auto spEventCpuImpl(event.m_spEventCpuImpl);
 
                     std::lock_guard<std::mutex> lk(spEventCpuImpl->m_Mutex);
+
                     if(!spEventCpuImpl->m_bIsReady)
                     {
                         spEventCpuImpl->m_bIsWaitedFor = true;
+
+                        auto const enqueueCount = spEventCpuImpl->m_enqueueCount;
```

Comment:
CI is crashing with errror `enqueueCount is unsed`. 

The error is wrong and we can workaround it by using `boost::ignore_unsed`

Reply:
solved by moving line into the #ifdef